backend/spherre/app/service/member.py

```
@@ -0,0 +1,28 @@
+from spherre.app.models.account import Member
+
+
+class MemberService:
+    @classmethod
+    def get_member_by_address(cls, address: str) -> Member | None:
+        """
+        Get a member by their address
+        """
+        return Member.query.filter_by(address=address).one_or_none()
+
+    def update_member_email(member_address: str, new_email: str) -> Member | None:
```

Pylint (3.3.7)

[error] Method 'update_member_email' should have "self" as first argument

(E0213)